### PR TITLE
[News] Update languages and tools on Feb 13, 2020

### PR DESCRIPTION
**Pull request by yykamei**
https://github.com/sider/runners/releases/tag/0.19.0

https://github.com/sider/runners/blob/ed742205b66fac5d60995a8f79bbf462c7928f9c/CHANGELOG.md#0190

### CODE REVIEW

File: docs/news/2020.md [hunk 1]
```diff
@@ -9,6 +9,45 @@ hide_title: true
...
+- Node.js - 12.14.0 to **12.14.1**
+- Go - 1.13.5 to **1.13.7**
+- PHP - 7.4.1 to **7.4.2**
+- Swift - 5.1.3 to **5.1.4**
+
+Updated Tools:
+
+- Ruby
+  - [RuboCop](../tools/ruby/rubocop.md)
+    - Support plugins maintained by rubocop-hq by default. See [this](https://github.com/sider/runners/issues/682) for more details
```
Comment (ybiquitous) [inline docs/news/2020.md]: [suggestion]

- Add a link to the "rubocop-hq" organization
- Make an issue link simple and clear

```suggestion
    - Support plugins maintained by [rubocop-hq](https://github.com/rubocop-hq) by default (issue [sider/runners#682](https://github.com/sider/runners/issues/682))
```

File: docs/news/2020.md [hunk 2]
```diff
@@ -9,6 +9,45 @@ hide_title: true
...
+    - Drop the "0.60.0" and the older versions of RuboCop. See [this](https://github.com/sider/runners/issues/131) for more details
```
Comment (ybiquitous) [inline docs/news/2020.md]:
```suggestion
    - Drop the 0.60.0 and older versions of RuboCop (issue [sider/runners#131](https://github.com/sider/runners/issues/131))
```

File: docs/news/2020.md [hunk 3]
```diff
@@ -9,6 +9,45 @@ hide_title: true
...
+  - [HAML-Lint](../tools/ruby/haml-lint.md) - 0.34.0 to **0.34.2** ([changes](https://github.com/sds/haml-lint/blob/master/CHANGELOG.md#0342)
+- Swift
+  - [SwiftLint](../tools/swift/swiftlint.md) - 0.38.0 to **0.38.2** ([changes](https://github.com/realm/SwiftLint/releases/tag/0.38.2)
+- Java
+  - [Checkstyle](../tools/java/checkstyle.md) - 8.28 to **8.29** ([changes](https://checkstyle.org/releasenotes.html#Release_8.29))
+  - [PMD](../tools/java/pmd.md) - 6.20.0 to **6.21.0** ([changes](https://pmd.github.io/pmd-6.21.0/pmd_release_notes.html))
+- PHP
+  - [PHP_CodeSniffer](../tools/php/codesniffer.md)
+    - squizlabs/php_codesniffer - 3.5.3 to **3.5.4** ([changes](https://github.com/squizlabs/PHP_CodeSniffer/releases/tag/3.5.4)
+    - escapestudios/symfony2-coding-standard - 3.10.0 to **3.11.0**
```
Comment (ybiquitous) [inline docs/news/2020.md]: [IMO] I think it enough that we say only about PHP CodeSniffer, not its plugins.

File: docs/tools/ruby/rubocop.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ hide_title: true
 
 | Supported Version           | Language   | Website                  |
 | --------------------------- | ---------- | ------------------------ |
-| 0.61.0+ (default to 0.79.0) | Ruby 2.5.7 | https://docs.rubocop.org |
+| 0.61.0+ (default to 0.79.0) | Ruby 2.6.5 | https://docs.rubocop.org |
 
 > **DEPRECATED**: Sider will drop the support of RuboCop 0.60.0 and older versions soon. Please upgrade to **0.61.0** and newer.
```
Comment (ybiquitous) [inline docs/tools/ruby/rubocop.md]: [question] Should we remain the deprecation warning yet?

File: docs/news/2020.md (hunk 3 above)
Comment (yykamei) [reply docs/news/2020.md]: I agree. Removed with 88d97f2

File: docs/tools/ruby/rubocop.md (hunk 4 above)
Comment (yykamei) [reply docs/tools/ruby/rubocop.md]: Thank you. Removed with cc45100

File: docs/news/2020.md [hunk 5]
```diff
@@ -9,6 +9,41 @@ hide_title: true
...
+  - [SwiftLint](../tools/swift/swiftlint.md) - 0.38.0 to **0.38.2** ([changes](https://github.com/realm/SwiftLint/releases/tag/0.38.2)
+- Java
+  - [Checkstyle](../tools/java/checkstyle.md) - 8.28 to **8.29** ([changes](https://checkstyle.org/releasenotes.html#Release_8.29))
+  - [PMD](../tools/java/pmd.md) - 6.20.0 to **6.21.0** ([changes](https://pmd.github.io/pmd-6.21.0/pmd_release_notes.html))
+- TypeScript
+  - [TSLint](../tools/javascript/tslint.md) - 5.20.1 to **6.0.0**
+- JavaScript
+  - [JSHint](../tools/javascript/jshint.md) - 2.10.3 to **2.11.0** ([changes](https://github.com/jshint/jshint/releases/tag/2.11.0))
+  - [ESLint](../tools/javascript/eslint.md)
+    - Drop the "4.19.1" and the older versions of ESLint. See [this](https://github.com/sider/runners/issues/715) for more details
```
Comment (ybiquitous) [inline docs/news/2020.md]:
```suggestion
    - Drop the 4.19.1 and older versions of ESLint ([sider/runners#715](https://github.com/sider/runners/issues/715))
```

**Comment by yykamei** [comment]
Thank you for your suggestions. All of them are pretty good. Thank you.

**Pull request merged**